tutorials/mnist_tutorial_fgsm_iterative.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import absolute_import
```

Comment:
I'm worried about the amount of code duplication from copy-pasting the whole tutorial every time we make a small change. If we need to change something in this code in the future, it will be a lot of places to copy the same change. Could you make a flag for the existing tutorial that determines whether to use fgsm or basic_iterative? If you don't want to do that much work, I also think it's OK to just not have a tutorial that uses iterative adversarial examples.